sql/slow-query.md

```diff
@@ -95,3 +95,25 @@ The SQL statement.
 Not all of the `SLOW_QUERY` statements are problematic. Only those whose `process_time` is very large will increase the pressure on the entire cluster. 
 
 The statements whose `wait_time` is very large and `process_time` is very small are usually not problematic. The large `wait_time` is because the statement is blocked by real problematic statements and it has to wait in the execution queue, which leads to a much longer response time.
+
+### `admin show slow` command
+
+In addition to the TIDB log file, you can identify slow queries by running the `admin show slow` command:
+
+    admin show slow recent N
+    admin show slow top [internal | all] N
```

Comment:
For code blocks and the following part, please refer to my comments here: https://github.com/pingcap/docs-cn/pull/922